pkg/catalog/install_test.go

```diff
@@ -166,4 +171,11 @@ status: {}
 			Expect(err).To(MatchError(`failed to parse url "invalid_1234%^": parse "invalid_1234%^": invalid URL escape "%^"`))
 		})
 	})
+
+	When("create-pr is set to true", func() {
+		It("can create a PR if the location is a git repository with change in it", func() {
+			err := catalog.CreatePullRequest(fakeScm, fakeGit)
+			Expect(err).ToNot(HaveOccurred())
+		})
+	})
```

Comment:
do we want to assert that the fake SCM and GIT clients are being invoked with the correct stuff?

Reply:
Yep. Note the TODO on the PR. :D I'm still writing the fake tests. There is a LOT. :D